### PR TITLE
Update dropdown on store page

### PR DESCRIPTION
**Pull request by nanaya**
Some hack involving floating the label is involved 🔧

### CODE REVIEW

File: resources/css/bem/input-container.less [hunk 1]
```diff
@@ -74,12 +71,33 @@
...
+
   &__label {
     display: flex;
     justify-content: space-between;
     margin: 0;
     color: var(--label-colour);
     font-size: @font-size--normal;
-    padding: var(--label-padding-base) var(--input-padding-base) 0;
+    padding: 5px var(--input-padding-base) 0;
+    position: absolute;
```
Comment (notbakaneko) [inline resources/css/bem/input-container.less:99]: can do with a comment on why it's `position: absolute`, also a note for the sizing of `--label-height` (or calculate it instead?)

**Review by notbakaneko — CHANGES_REQUESTED** [review]

File: resources/css/bem/input-container.less [hunk 2]
```diff
@@ -3,7 +3,9 @@
 
 .input-container {
   --label-font-size: @font-size--normal;
-  --label-height: calc(var(--label-font-size) + var(--input-padding-base) + 5px);
+  --label-height: calc(
```
Comment (notbakaneko) [inline resources/css/bem/input-container.less:6]: I think this should be `font-size * 1.25 (the line-height) + padding base` tbh 🤔

Comment (nanaya) [reply resources/css/bem/input-container.less:6]: eh close enough 🤷

**Review by notbakaneko — APPROVED** [review]

**Comment by nanaya** [comment]
wait no, that was with wrong padding - it should be 5px both sides

**Comment by notbakaneko** [comment]
wut

**Review by notbakaneko — APPROVED** [review]

**Pull request merged**